test/unit/int_alma/scripts/helpers/almaRefundHelperTest.js

```diff
@@ -31,8 +30,8 @@ describe('almaRefundHelpers', function () {
     });
 
     it('partial refund', function () {
+        order.custom.almaRefundedAmount = 0;
         almaRefundHelpers.refundPaymentForOrder(order, 10);
-        assert.equal(order.custom.ALMA_Refunded, true);
```

Comment:
can we assert for `order.custom.almaRefundedAmount` instead of just removing this ?

Could we also have a test for the other custom attribute to ensure it goes back to 0 after a refund and everything ?